src/main/java/org/codehaus/plexus/components/io/resources/PlexusIoFileResourceCollection.java

```diff
@@ -158,8 +159,13 @@ private void addResources( List<PlexusIoResource> result, String[] resources )
         {
             String sourceDir = name.replace( '\\', '/' );
             File f = new File( dir, sourceDir );
+            Path p = f.toPath();
+            if ( isFollowingSymLinks() )
```

Comment:
This is `true` by default `isFollowingSymLinks`. I'm a bit concerned that this change would change the behavior of existing applications. I know the existing behavior is odd, but it was existing for quite some time and maybe there are applications that depend on it. Do you think adding additional flag would made sense? In general I hate fix bug kind of flags, but this code was here for quite some time and maybe there is code out there depending on `PlexusIoFileResourceCollection` not following symlinks by default.